scripts/buildAlloy.cjs

```diff
@@ -30,18 +30,11 @@ const execute = (command, options) => {
 };
 
 const getBrowserListFromRcFile = () => {
-  const lastPath = ["./src/lib/.browserslistrc", ".browserslistrc"].reduce(
-    (acc, curr) => {
-      if (fs.existsSync(`${__dirname}/../${curr}`)) {
-        acc = curr;
-      }
-
-      return acc;
-    },
+  const babelConfigFile = "babel.config.json";
```

Comment:
The browsers are added in the babel.config.json file. The build script is now fetching the browsers from the Babel config file.

If we want to no longer have separate .browserslistrc files (we had one for `src/lib` and one for `src/view`) we might just add one `.browserslistrc` file at the root.